src/Refactor/Xml/XmlFormatter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace DemandwareXml\Refactor\Xml;
+
+use DateTimeInterface;
+
+// Converts values into strings suitable for XML output.
+class XmlFormatter
```

Comment:
This class attempts to wrap up the ways we handling casting things strings for use as XML output.